openweather/src/main/java/actor/SignInTester.java

```diff
@@ -0,0 +1,21 @@
+package actor;
+
+import action.SignInAction;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class SignInTester {
```

Comment:
Don't break too small like this, ask yourself what is different between testing SignIn and Search Page?